index.html

```diff
@@ -206,5 +206,78 @@ <h4>Skills</h4>
         
     </section>
 </section>
+
+<footer>
+        <div class="footspace"></div> 
+        <div class=" box namet"><h2>Get Started</h2></div>
+        <div class=" box introt"><p>Lorem ipsum dolor sit amet,
+             consectetur adipiscing elit, sed do eiusmod tempor
+              incididunt ut labore et dolore magna.</p></div>
+<form  action="https://formspree.io/f/xbjwlnpa" method="post">
+   <div class="fooc">
+        <fieldset>
+            <div class="field">
+            <legend>Your name</legend>
+            <input type="text" name="username" maxlength="30" placeholder="Last name First name" required>
+        </div>
+        </fieldset>
+    </div>
+    <div class="fooc">
+        <fieldset>
+            <div class="field">
+            <legend>Your email</legend>
+            <input type="text" name="email" placeholder="someone@domain.com" required>
+        </div>
+        </fieldset>
+    </div>
+    <div class="fooc">
+        <fieldset>
+            <div class="field">
+                <textarea name="comment" maxlength="500" placeholder="Enter your message here..." required></textarea>
+            </div>
+        </fieldset>
+    </div>
+        <button value="submit">
+            Start collaboration</div></button> 
```

Comment:
- [x] Please, stick to the design template by centering the button text 😸

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/149049112-d7c1cc0b-6abd-4f55-9299-6f0f8ef8921a.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/149049115-d109472d-5eb8-480c-9ec7-b8f13d7a43ed.PNG)